docker-compose.yml

```diff
@@ -1,13 +1,13 @@
-version: "3.1" 
+version: "3.1"
 services:
   glm_test:
-    image: wdwatkins/glm3r:v0.3
+    image: aapplingusgs/glm3r:v0.4
     build:
       context: .
     ports:
       - "8787:8787"
     volumes:
-      - data:/home/rstudio/glm3_test                                                                                              
+      - data:/work/glm3_test
```

Comment:
Hmm, looking at this again, I see that I didn't make a comprehensive decision about the `glm3_test` directory - I define it as a volume in this line, but I removed the `mkdir` call for this directory in the Dockerfile. I admit to not really understanding how/whether this data volume is necessary - do you have any advice, David? Could we just remove the volume entirely, or is there a reason to keep it? I think on Docker this directory makes it possible to test the container without custom-binding an external directory, but with Shifter's merging of access to the image and local file systems, and removal of access (at least on Denali) to the /home/rstudio directory, it seems a lot less necessary or useful....

Reply:
Yeah, to my knowledge this is superfluous if we don't ever use this image in Docker form, since Singularity (and I'm guessing Shifter as well) bind mount by default.